pages/admin/[reviewEditId].tsx

```diff
@@ -10,14 +10,24 @@ import AdditionalEditCard from "../../components/AdditionalEditCard";
 import OutsideClickHandler from 'react-outside-click-handler';
 import { getSession } from "next-auth/react";
 import HamburgerMenu from "../../components/HamburgerMenu";
+import AccountPanel from "../../components/AccountPanel";
+import { Session } from "next-auth";
+import { UserModel } from "../../models/user";
 
 
 
 
 export async function getServerSideProps(context: any) {     //fetch data from the server, runs on the server
 
     const session = await getSession(context)
-    if (!session) return {redirect :{permanent: false,destination :"/login"}}         //go back to sign in page
+    if (!session) return { redirect: { permanent: false, destination: "/login" } }         //go back to sign in page
+    await mongoose.connect(process.env.MONGODB_URL as string)
+    const thisUser = await UserModel.findOne({ email: session.user.email })    //double check just in case session exists, but user unauthorised
+
+    if (!thisUser) {
+        return { redirect: { permanent: false, destination: "/login" } }        //go back to sign in page
```

Comment:
Maybe go back to `/login?unauthorised=true` so the session gets ended?

Reply:
good idea